content/en/security/application_security/enabling/serverless.md

```diff
@@ -168,6 +221,12 @@ To install and configure the Datadog Serverless Framework plugin:
      DD_EXPERIMENTAL_ENABLE_PROXY: true
      AWS_LAMBDA_EXEC_WRAPPER: /opt/datadog_wrapper
    ```
+4. For **Node** and **Python** functions only, double-check that the function's handler is set correctly
+    - **Node**: Set your function’s handler to /opt/nodejs/node_modules/datadog-lambda-js/handler.handler. 
```

Comment:
- **Node**: Set your function’s handler to `/opt/nodejs/node_modules/datadog-lambda-js/handler.handler`.